### PR TITLE
chore(frontend): Remove unused prop of TokenUiGroup

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We remove the props that are not needed from `TokenUiGroup` and we keep only the needed ones.

### CODE REVIEW

**Comment by AntonioVentilii-DFINITY** [comment]
[OISY-69]

File: src/frontend/src/tests/lib/utils/token-card.utils.spec.ts [hunk 1]
```diff
@@ -30,7 +22,7 @@ describe('mapHeaderData', () => {
 			decimals: ICP_TOKEN.decimals,
 			icon: ICP_TOKEN.icon,
 			network: ICP_NETWORK,
-			oisyName: { oisyName: BTC_MAINNET_TOKEN.name },
+			oisyName: { oisyName: ICP_NETWORK.name },
```
Comment (peterpeterparker) [inline src/frontend/src/tests/lib/utils/token-card.utils.spec.ts:25]: It's fine but maybe for a safer test we should pick another token. The ICP network and token have the same name.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/tests/lib/utils/token-card.utils.spec.ts (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/tests/lib/utils/token-card.utils.spec.ts:25]: right! I will use SEPOLIA_ETH in another PR:

- PR #

**Pull request merged**